pyproject.toml

```diff
@@ -30,7 +30,7 @@ python-dateutil = "^2.8.2"
 tiktoken = ">=0.5.1"
 nltk = "^3.8.1"
 litellm = "^1.37.14"
-
+sentence-transformers = "3.0.1"
```

Comment:
sentence-transformers depends on transformers and torch.  We have those under a separate sub-dependency.  Do we need sentence-transformers here or can we shuffle it to the other section?

Reply:
I think I can actually remove it.  We were using it for a stab at anomaly detection but I don't think we'll ship with that this time around because it requires another model to be shipped with core which we don't want.